spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/security/AuthenticationAuditListener.java

```diff
@@ -73,6 +76,9 @@ else if (this.webListener != null && this.webListener.accepts(event)) {
 		else if (event instanceof AuthenticationSuccessEvent successEvent) {
 			onAuthenticationSuccessEvent(successEvent);
 		}
+		else if(event instanceof LogoutSuccessEvent logoutSuccessEvent) {
```

Comment:
Please check the formatting. You can run `./gradlew spring-boot-project:spring-boot-actuator:format` to correct it automatically.